### ISSUE TITLE
About trainning pose estimator 

### ISSUE THREAD

**Issue by dingshenglan** [jimmyyhwu/pose-interpreter-networks#12]
Hello, I changed the model in pose estimator from  resnet18 to  resnet50, I use the points loss, but when I train the estimator on th mask dataset, the orientation loss does not converge，could you give me some suggestion？
![image](https://user-images.githubusercontent.com/34231078/56078641-fb511100-5e1c-11e9-9bcd-8bac773f74c1.png)
![image](https://user-images.githubusercontent.com/34231078/56078642-fe4c0180-5e1c-11e9-9361-346168bec827.png)
![image](https://user-images.githubusercontent.com/34231078/56078643-0015c500-5e1d-11e9-81e7-800a13289c46.png)

**Comment by jimmyyhwu** [comment]
I have not tried using ResNet-50, you may have to change the hyperparameters for the bigger model. From your plots it looks like the learning rate may be too high.

**Comment by dingshenglan** [comment]
> 我没有尝试使用ResNet-50，您可能需要更改较大型号的超参数。从您的情节来看，学习率可能太高了。

Ok，I will use smaller learning rate. Thanks verymuch :)
